src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -178,6 +178,7 @@ class AuthScreens extends React.Component {
                 // a header will briefly open and close the keyboard and crash Android.
                 // eslint-disable-next-line react/jsx-props-no-multi-spaces
                 keyboardHandlingEnabled={false}
+                detachInactiveScreens={false}
```

Comment:
When I don't set this prop, chat window view in the background doesn't work for modal views on web. We just have gray background.

Reply:
Interesting, @tugbadogan how did you determine that setting that prop to false would fix the issue? Just wondering if there are alternate solutions since this has the side effect @marcaaron mentioned above.

Reply:
I found this by accident while looking at the documentation. But, we don't need this anymore as I fixed the issue by upgrading react-native-screens package version to >3.0.0 as required by react-navigation 6.x and used `presentation: 'transparentModal'` for modal screen options.